src/devtools/client/debugger/src/actions/pause/paused.ts

```diff
@@ -79,8 +82,8 @@ export function paused({
   return async function (dispatch, getState, { ThreadFront }) {
     dispatch(pauseRequestedAt(executionPoint));
 
-    if (!isPointInLoadedRegion(getState(), executionPoint)) {
-      return;
+    if (!isPointInLoadingRegion(getState(), executionPoint)) {
+      dispatch(failedToCreatePause(executionPoint, ""));
```

Comment:
If we've ended up in a state where we are truly not going to load this point, we should not leave it marked as "loading"